tutorial/20_recipes/010_reuse_best_trial.py

```diff
@@ -0,0 +1,92 @@
+"""
+.. _reuse_best_trial:
+
+Re-use the best values
+==========================
+
+You can re-evaluate the objective with the best hyperparameters again
+after the hyperparameter optimization.
+Example,
+
+- You have found good hyperparameters with Optuna, and want to run a similar `objective` function using the best hyperparameters found so far to further analyze the results, or
+- You have optimized with Optuna using a partial dataset to reduce training time. After the hyperparameter tuning, you want to train the model using the whole dataset with the best hyperparameter values found.
+
+Optuna provides an interface to re-evaluate the objective function with the current best hyperparameter values.
```

Comment:
```suggestion
``study.best_trial``
 provides the interface to re-evaluate the objective function in these cases.
```

Reply:
I use :class:`~optuna.study.Study.best_trial` rather than `study.best_trial`, to link the informative documentation.